android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -235,8 +235,14 @@ public void sendToNotificationCentre(Bundle bundle) {
             bundle.putBoolean("userInteraction", true);
             intent.putExtra("notification", bundle);
 
+            Intent summaryIntent = new Intent(context, intentClass);
+            intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
+            bundle.putBoolean("userInteraction", true);
+
```

Comment:
`intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);` <--- but this was supposed to be for `summaryIntent`

Also didn't `putExtra` into the bundle like:
`summaryIntent.putExtra("notification", bundle);`